### ISSUE TITLE
Arduino IDE enhancement request

### ISSUE THREAD

**Issue by PickyBiker** [arduino/Arduino#9010]
The error messages that display in the bottom of the Arduino IDE are currently fixed at a red/orange color for the text and black for the background. 
On my system that is difficult to read. 
Please add a feature to allow adjusting those colors in the preferences.txt file.

**Comment by per1234** [comment]
Closing as duplicate of https://github.com/arduino/Arduino/issues/6454.

Note that it is possible to change these colors by editing the theme. You'll find documentation here:
https://github.com/per1234/ThemeTest

**Comment by per1234** [comment]
Closing as duplicate of https://github.com/arduino/Arduino/issues/6454.

Note that it is possible to change these colors by editing the IDE's theme. You'll find documentation here:
https://github.com/per1234/ThemeTest
